auction-domain/test/lib/auction/domain/auction_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+require 'auction/domain/auction'
+require 'auction/domain/item'
+
+describe Auction do
+  it 'requires a deadline' do
+    proc { Auction.new }.must_raise Errors::InvalidDeadlineError
+  end
+
+  it 'requires a list of items' do
+    proc { Auction.new(deadline: Time.new(2015, 01, 01)) }.must_raise Errors::EmptyAuctionItemsError
```

Comment:
Why to not follow the same naming pattern of `InvalidDeadlineError`? What do you think of using `InvalidItemsError`